kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/KafkaProxyReconciler.java

```diff
@@ -444,6 +450,29 @@ public List<EventSource<?, KafkaProxy>> prepareEventSources(EventSourceContext<K
                 buildKafkaProxyIngressEventSource(context));
     }
 
+    private static Optional<AllowDeny<String>> buildProtocols(@Nullable Protocols protocols) {
+        return Optional.ofNullable(protocols)
+                .map(p -> new AllowDeny<>(p.getAllow(), new HashSet<>(p.getDeny())));
+    }
+
+    private static Optional<AllowDeny<String>> buildCipherSuites(@Nullable CipherSuites cipherSuites) {
+        return Optional.ofNullable(cipherSuites)
+                .map(c -> new AllowDeny<>(c.getAllow(), new HashSet<>(c.getDeny())));
+    }
+
+    private static Optional<ServerOptions> buildTlsSeverOptions(@Nullable TlsClientAuthentication clientAuthentication) {
+        var knownNames = Arrays.stream(TlsClientAuth.values())
+                .map(TlsClientAuth::name)
+                .collect(Collectors.toSet());
+
+        return Optional.ofNullable(clientAuthentication)
+                .map(TlsClientAuthentication::getValue)
+                .filter(knownNames::contains)
+                .map(TlsClientAuth::valueOf)
+                .filter(Predicate.not(TlsClientAuth.NONE::equals))
```

Comment:
following #2135 this should be either changed to `REQUIRED` or removed. I'd be in favour of pushing through an explicit `REQUIRED` into the proxy config if the user has specified it in the CR, even though it's default behaviour.